package.json

```diff
@@ -29,14 +34,15 @@
   "main": "./chai.js",
   "scripts": {
     "prebuild": "npm run clean",
-    "build": "npm run build:esm",
-    "build:esm": "esbuild --bundle --format=esm --keep-names --outfile=chai.js index.js",
+    "build": "npm run build:ts && npm run build:esm",
```

Comment:
Why are we doing both? Is it to check for type errors? If yes, can we create a `"lint:types": "tsc --noEmit"` task and run that in the existing `lint` task?